python/vowpalwabbit/pyvw.py

```diff
@@ -1705,71 +1614,24 @@ def initialize_col_type(self, instance, colname):
                 min_value=self.min_value,
             )
 
-    def generate_error(self, instance, valid_type, valid_value):
-        """Generate message error for the attribute
-
-        Parameters
-        ----------
-        instance: object
-            The managed class.
-        valid_type: bool
-            True if the type if valid, False otherwise.
-        valid_value: bool
-            True if the value range if valid, False otherwise.
-
-        Raises
-        ------
-        TypeError
-            If the type is not valid.
-        ValueRange
-            If the value range is not valid.
-        """
-        class_name = type(instance).__name__
-
-        if not valid_type:
-            raise TypeError(
-                "In '{class_name}', when {from_df_false}, argument '{attribute_name}' should be either of the following type(s): {type_name_msg}.".format(
-                    class_name=class_name,
-                    from_df_false=self.attribute_name + "_from_df=False",
-                    attribute_name=self.attribute_name,
-                    type_name_msg=str(
-                        [x.__name__ for x in self.expected_type]
-                    )[1:-1],
-                )
-            )
-
-        if not valid_value:
-            raise ValueError(
-                "In '{class_name}', argument '{attribute_name}' must be >= {min_value}.".format(
-                    class_name=class_name,
-                    attribute_name=self.attribute_name,
-                    min_value=self.min_value,
-                )
-            )
-
 
 class SimpleLabel(object):
     """The simple label type for the constructor of DFtoVW."""
 
     name = AttributeDescriptor("name", expected_type=(int, float))
```

Comment:
semirelated to this PR but @lalo what do you think of changing the `name` arg into `label` (and same for MulticlassLabel, etc.)

Reply:
I think that stands with our current docs, if you refer to https://github.com/VowpalWabbit/vowpal_wabbit/wiki/Input-format you can see that the terminology uses 'label' even for multiclass.

Reply:
Ok i've just change it !